buildpipeline/pipeline.json

```diff
@@ -20,7 +20,7 @@
           "Parameters": {
             "PB_DistroRid": "rhel.7.2-x64",
             "PB_DockerTag": "rhel7_prereqs_2",
-            "PB_AdditionalBuildArguments":"-PortableBuild=true",
+            "PB_AdditionalBuildArguments":"-PortableBuild=true -strip-symbols",
```

Comment:
Hunting around, it appeared that symbol stripping applied only to Release builds.  The alternative would be to have different default behavior between Debug / Release.  I modeled the behavior after how this is being handled in coreclr